### ISSUE TITLE
Move protos to separate repo?

### ISSUE THREAD

**Issue by masomel** [citp/coniks-ref-implementation#41]
Maintaining the protobufs in a separate repo would make it easier to ensure that both CONIKS implementations use compatible protobuf messages. Any changes to the protobufs could then be easily propagated to both implementations. This repo would contain the compiled protobufs for each supported language, which can then be imported by each implementation.

**Comment by vqhuy** [comment]
Yeah, I think so, too. I also think that we can adopt protobufs into our storage backend hook. It would replace the `Deserialize` and `Serialize` problem we're facing.

**Comment by masomel** [comment]
That's an interesting point, this is definitely something we should consider.
